book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(rental)
+    @rental << rental
+  end
```

Comment:
- [ ] It would be better if you add person and date into the parameters, create the rental and then push into the rentals array.
```
  def add_rental(person, date)
    Rental.new(date, self, person)
    @rentals.push(rental)
  end
```